tests/puppeteer/campaigns/utils/files.js

```diff
@@ -9,4 +9,14 @@ module.exports = {
   async deleteFile(pathToFile) {
     if (fs.existsSync(pathToFile)) fs.unlinkSync(pathToFile);
   },
+
+  /**
+   * Check File existence
+   * @param downloadPath
+   * @param fileName
+   * @return boolean, true if exist, false if not
+   */
+  async isFileExist(downloadPath, fileName) {
+    return fs.existsSync(`${downloadPath}/${fileName}`);
```

Comment:
is it really useful to create a function to call a function :thinking: 
it's like doing this
```
function consolelog(downloadPath, fileName) {
  console.log(`${downloadPath}/${fileName}`);
}
```